src/target/llvm/codegen_cpu.cc

```diff
@@ -466,13 +466,14 @@ llvm::Value* CodeGenCPU::CreateCallExtern(Type ret_type, String global_symbol,
   llvm::FunctionType* ftype = llvm::FunctionType::get(GetLLVMType(ret_type), arg_types, false);
   // Check if it is available in global function table as injected function.
 
+  llvm::Function* f = module_->getFunction(MakeStringRef(global_symbol));
```

Comment:
Instead of exposing `llvm::Function* f` outside the scope, could we instead replace `auto ext_callee = f;` on line 490 with `auto ext_callee = callee;`?  The current change would allow compilation, but would skip the optional pointer-cast of the callee.

Reply:
Or, alternatively, I think we could simplify it by returning `builder_->CreateCall(ftype, callee, arg_values);`.  This method is available as far back as 3.7.0 ([commit link](https://github.com/llvm/llvm-project/commit/3a6a69c535066c7cadc434d693eeeb5e5886abb1)), and since TVM requires LLVM 4.0 or higher, we would avoid needing the `#if` directive altogether.

Reply:
@Lunderberg @quic-sanirudh Thanks for your guidance. I have corrected the patch, could you recheck the code change again?